src/nanoarrow/common/array_test.cc

```diff
@@ -3380,29 +3423,17 @@ void TestGetFromBinaryView(BuilderClass& builder) {
   EXPECT_EQ(ArrowArrayViewIsNull(&array_view, 2), 1);
   EXPECT_EQ(ArrowArrayViewIsNull(&array_view, 3), 0);
 
-  auto string_view = ArrowArrayViewGetStringUnsafe(&array_view, 3);
```

Comment:
A bit of a refactor mixed in, but I didn't want to keep repeating this pattern of testing both String/Byte get functions for all Ts

Reply:
Also generally curious - why do we keep the test suite tied to C++11? Is that to let downstream libraries run the tests without imposing a more recent standard?